### DISCUSSION TITLE
Banned weapons

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12404] [Ideas and suggestions]
Outpost security should attempt to arrest anyone with a weapon, plasma cutter or crowbar in hand due to the fact we have no lawful reason to have it out unless there is an event that makes hostiles appear. NPCs should also report you if they see you with them. Also having a diving suit in the outpost should cause security to follow you. Also if security is trying to arrest or kill any of the crew the hatch down to the submarine should lock until security is idle.

**Comment by VaelophisNyx** [comment]
I disagree with some of this

namely the diving suit one. It's harmless enough.
That and the arbitrary lock would be....Hard to do? And kinda pointless?
